samples/client/petstore/python/petstore_api/models/outer_enum.py

```diff
@@ -30,6 +30,8 @@ class OuterEnum(object):
     APPROVED = "approved"
     DELIVERED = "delivered"
 
+    allowable_values = [PLACED, APPROVED, DELIVERED]  # noqa: E501
```

Comment:
@slash-arun is this what you intend?
These allowable_values are the keys or accessors on this enum class, not the accepted values

Reply:
Also, do you want this ability on api endpoints too?

If so could the python-experimental generator meet your needs? It has this feature for models:
https://github.com/OpenAPITools/openapi-generator/blob/master/samples/client/petstore/python-experimental/petstore_api/models/enum_class.py#L48
and for api_endpoints:
https://github.com/OpenAPITools/openapi-generator/blob/master/samples/client/petstore/python-experimental/petstore_api/api/fake_api.py#L819

Reply:
These will evaluate to allowed values only right ['placed', 'approved', 'delivered'] ?

Could you let me know if python-experimental is fairly stable ?

Reply:
I'm not sure, you will have to import your python package and see if it works that way. My python enum game is not strong.

So I made python-experimental recently as a fork of the python client to add a new additionalproperties feature and to structure the data better so users can automatically have features like this one that we are discussing. It may change more quickly than the python client, but as long as you version pin to the version that you want it should be stable.
It has not existed for very long, maybe a couple of months. It also is structured in a way where we will more easily be able to implement oneOf, allOf, anyOf inheritance in the generator.

Reply:
Thanks for the detailed explanation @spacether :) I am just a bit concerned about using it in production at this point, to be honest.

Reply:
No worries. It's your call